src/firefly/js/ui/UploadTableSelector.jsx

```diff
@@ -127,8 +127,8 @@ export function CenterColumns({lonCol,latCol, sx, cols, lonKey, latKey, openKey,
              <Typography component='div'>
                  <Stack {...{direction:'row', alignItems:'baseline', spacing:1}}>
                      <span>{headerTitle}</span>
-                     <Typography color='warning'>{(lonCol || latCol) ? `${lonCol || 'unset'}, ${latCol || 'unset'}` : 'unset'}</Typography>
-                     <Typography level='body-sm'>{`${headerPostTitle}`}</Typography>
+                     <Typography level={'body-md'} sx={{fontWeight:'normal'}}>{(lonCol || latCol) ? `${lonCol || 'unset'}, ${latCol || 'unset'}` : 'unset'}</Typography>
+                     <Typography level='body-sm' sx={{fontWeight:'normal'}}>{`${headerPostTitle}`}</Typography>
```

Comment:
Why did you have to adjust fontWeight directly?  Typography level is suppose to do that.  Having a finite number of level is so that we will not have too many variation of font-size/weight.  Similar concept to variant and color.

Reply:
yes, I know, but the `Typography` is inside a button which is forcing it to bold. So to actually get `body-md` it had to force the `fontWeight`. I was a little surprised by that.